pkg/esp8266_sdk/bootloader/sdkconfig_default.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Generated by ./update_mk.sh, don't modify directly.
```

Comment:
@iosabi Is this file really needed as part of the repository or could the initial version also be generated if not existing?

Reply:
@iosabi ping

Reply:
I think I wrote a comment somewhere else about this. I looked into it but the issue is that to generate this file you need to have both the host toolchain (to run the "make defconfig") and the target esp8266 toolchain. I don't think that any of these settings are relevant, they are just used by the sources compiled and must be defined but the defaults are ok. The interesting ones are in `pkg/esp8266_sdk/bootloader/sdkconfig.h`, and if you add more there (tied to some RIOT config) the script will remove them from the `sdkconfig_default.h`. I think generating it when building RIOT was too technically complex to warrant it since a static file is enough. For the .mk file with the list of sources I'm not sure it is possible or easy.

Reply:
@iosabi Please consider to merge and push my changes to your branch. We can try again to make Murdock happy.

Reply:
@iosabi With PR #17457, the concept of compiling the boot loader from package sources works, including adding the binary to `TEST_EXTRA_FILES`. Murdock is satisfied. So the changes I provided in https://github.com/iosabi/RIOT/pull/1 should work as well.
If it works for both platforms, we could move some stuff to `cpu/esp_common`.